R/boost_tree.R

```diff
@@ -371,12 +372,13 @@ recalc_param <- function(x, counts, denom) {
 
 maybe_proportion <- function(x, nm) {
   if (x < 1) {
-    msg <- paste0(
-      "The option `counts = TRUE` was used but parameter `", nm,
-      "` was given as ", signif(x, 3), ". Please use a value >= 1 or use ",
-      "`counts = FALSE`."
+    cli::cli_abort(
+      "The option `counts = TRUE` was used but parameter {.arg nm},
+      was given as signif(x, 3).Please use a value >= 1
+      or set {.code counts = FALSE}."
```

Comment:
```suggestion
    cli::cli_abort(
      "The option `counts = TRUE` was used but parameter {.arg nm},
      was given as signif(x, 3). Please use a value >= 1
      or set {.code counts = FALSE}."
```

I'm seeing package tests failing related to this test:

```
Error in `maybe_proportion(x, nm)`: The option `counts = TRUE` was used but parameter `nm`, was given as signif(x, 3).Please use a value >= 1 or set `counts = FALSE`.
```

Could you transition that test from using `expect_error()` to `expect_snapshot(error = TRUE, ...)`?